README.md

```diff
@@ -10,11 +10,12 @@
 
 ## Live Demo
 
+There is the link to the online version
 https://sergskol.github.io/portfolio
 
```

Comment:
- [ ] You did a good job but when I try to open with `live-server` then it renders me on some page that says `404`. please fix this issue and submit it again.
### Error
![dd](https://user-images.githubusercontent.com/81727988/194828705-ea85e12d-2031-4da7-97e6-fe2fe412bcb3.png)

Reply:
Thank you! One letter typo https://sergskol.github.io/Portfolio/  -- the capital P in Portfolio